dbsnap_verify/__init__.py

```diff
@@ -0,0 +1,154 @@
+from time import time
+
+import datetime
+
+import json
+
+import boto3
+s3 = boto3.client("s3")
```

Comment:
Does this need region_name, or does lambda set AWS_REGION automatically in the environment?

Reply:
Um, good catch. I need to use the region passed from cloudwatch event rule trigger when connecting to RDS, but I'm not sure about s3 - I though s3 was global.